### DISCUSSION TITLE
Not seeing extended stock options in Stock menu

### DISCUSSION THREAD

**Discussion by wafflecrate** [inventree/InvenTree#5516] [Q&A] [answered]
Hello-

I'm running Inventree 0.12.6 and have a user account that has all Superuser permissions.

The Documentation for Stock Adjustments shows all sorts of options for the Stock menu: 

1. Add Stock
2. Remove stock
3. Count stock
4. Move stock
5. Merge stock
6. Order stock, etc.

![stock_options](https://github.com/inventree/InvenTree/assets/67816847/0d0b0d26-a717-4e41-afd6-6abba82db5a6)

But when I go into that same menu on my live Inventree I only get two options: 

1. Count part stock
2. Transfer part stock
![stock menu](https://github.com/inventree/InvenTree/assets/67816847/e58e839f-9717-483a-bd00-20e2a9dd748e)

Any suggestions as to why this is happening? I've double checked my user permissions and my User account is part of the "Managers" group that has all VACD privileges. I must be doing something wrong, here.
![Groups](https://github.com/inventree/InvenTree/assets/67816847/d8f2c8da-a245-4e3e-b27d-12803b6e66ef)
![Permissions](https://github.com/inventree/InvenTree/assets/67816847/6cf40eae-6c40-48a2-a7f8-be2ae0a1c095)

I'm also unable to add in pricing information for my Stock items so I'm wondering if it's all related to the same issue?

**Comment by SchrodingersGat** [comment]
The second screenshot you have added there does not seem to line up with what you are describing. It is not of the same view even?

**Reply by wafflecrate** [reply]
Yes just noticed that and corrected it in the original post.

**Reply by SchrodingersGat** [reply]
Ok, still not comparing the same screens though. First screenshot is from the "stock table" view. Second is from the "part detail" view - there are different actions available when comparing those screens. This is completely expected.

**Reply by wafflecrate** [reply]
I see what you're saying- I'm in the wrong section! So how do I get into the "stock table" view? Because with all of my clicking around I always seem to end up in the "part detail" view which obviously has much fewer stock options.

edit: Also realized that I am running Inventree 0.11.0.

**Reply by SchrodingersGat** [reply, marked as answer]
![image](https://github.com/inventree/InvenTree/assets/10080325/4d2cb153-ff11-4123-b329-f649d0d37fa7)

**Reply by wafflecrate** [reply]
Thanks for the screenshot. So what I actually need to do is click the number under the "Stock" table, THEN click the "Stock Adjustment Options". I learned something new.

![stock table](https://github.com/inventree/InvenTree/assets/67816847/f5aa5cfb-a1d3-4b64-9e7d-ed96e76df811)
![Additional](https://github.com/inventree/InvenTree/assets/67816847/37f4aa3a-4544-4ea7-922d-1d4facfd5125)
